python/tank/authentication/authentication_session_data.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2017 Autodesk.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+"""
+Authentication Session abstraction.
+"""
+
+
+class AuthenticationSessionData(object):
+    """
+    Holds session information.
+
+    A session object holds information coming from both the toolkit current session
+    and from the QWebView used to login/renew session from our SSO enabled site.
+
+    Attributes:
+        cookies:            A string of the base64 encoded json string of raw
+                            cookies.
+        crsf_value:         A string for the crsf value.
+        csrf_key:           A string for the crsf key.
+        error:              A string which describe the error encountered.
+        host:               A string for the hostname.
+        product:            A string for the product (used at login).
+        session_expitation: An int in seconds for the UTC time of expiration.
+        session_id:         A string for the session id.
+        user_id:            A string for the user id.
+    """
+
+    def __init__(self, settings={}):
+        """
+        Constructor.
+
+        Args:
```

Comment:
This is a shared file between RV and Toolkit...

For the Toolkit convention for documentation, could you point the reference to me ?